BookAPI/Models/Author.cs

```diff
@@ -0,0 +1,83 @@
+﻿namespace BookAPI.Models
+{
+    using System.ComponentModel.DataAnnotations;
+
+    /// <summary>
+    /// Author entity
+    /// </summary>
+    public class Author
+    {
+        /// <summary>
+        /// The global count for id autoincrement
+        /// </summary>
+        private static int _globalCount;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Author"/> class.
+        /// </summary>
+        /// <param name="fullname">The fullname of author.</param>
+        /// <param name="country">The country.</param>
+        public Author(string fullname, string country)
+        {
+            this.FullName = fullname;
+            this.Country = country;
+            this.Id = ++_globalCount;
+        }
+
+        /// <summary>
+        /// Gets or sets the identifier.
+        /// </summary>
+        /// <value>
+        /// The identifier.
+        /// </value>
+        [Required(ErrorMessage = "Id is always required!")]
+        [Range(1, long.MaxValue, ErrorMessage = "Id should be natural number!")]
+        public long Id { get; set; }
```

Comment:
A question that Alexander would have asked in this case: Why do you use long instead of ulong if you know that your values will not be negative?